ghc.nix

```diff
@@ -165,6 +175,26 @@ let
         ];
         librarySystemDepends = depsSystem;
       });
+
+  findNoteDef = writeShellScriptBin "find_note_def" ''
+    ret=$(rg  --no-messages --vimgrep -i --engine pcre2 "^ ?[{\\-#*]* *\QNote [$1]\E\s*$")
```

Comment:
I think it would be preferable to refer directly to the nixpkgs version here eg `${pkgs.lib.getExe pkgs.ripgrep}`

Reply:
Oh, good catch. This is an oversight from my local experiments... What's the difference between `${pkgs.ripgrep}/bin/rg` and `${pkgs.lib.getExe pkgs.ripgrep}`?